cookies/README.md

```diff
@@ -0,0 +1,2 @@
+This directory contains tests for
+[Leave Secure Cookies Alone](https://tools.ietf.org/html/draft-ietf-httpbis-cookie-alone-01).
```

Comment:
Seems like https://github.com/w3c/web-platform-tests/blob/master/cookies/path/match.html covers a bit more than the IETF draft you linked. I think this is a reasonable first step and we can rely on @mikewest's guidance for the next step 😸